<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolynomialRoots</class>
 <widget class="QMainWindow" name="PolynomialRoots">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>395</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>286</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>286</width>
    <height>395</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Miejsca zerowe wielomianu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="coeffiicientsLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>35</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Współczynniki (oddzielone spacjami):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coefficientsEditBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>256</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="precisionLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>80</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dokładność:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="precisionEditBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>100</y>
      <width>256</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.001</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="methodGroupBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>130</y>
      <width>256</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Metoda</string>
    </property>
    <widget class="QRadioButton" name="bisectionRadioButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>96</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>połowienia</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="newtonRadioButton">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>20</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Newtona</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="beginEditBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>205</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
   </widget>
   <widget class="QLabel" name="beginLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>185</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Początek przedziału:</string>
    </property>
   </widget>
   <widget class="QLabel" name="endLabel">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>185</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Koniec przedziału:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="endEditBox">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>205</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>270</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wynik:</string>
    </property>
   </widget>
   <widget class="QLabel" name="windowLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Miejsca zerowe wielomianu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="computeButton">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>235</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Oblicz</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultTextBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>290</y>
      <width>256</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>286</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <widget class="QMenu" name="menuPrzyklad">
     <property name="title">
      <string>Przyklad</string>
     </property>
     <addaction name="actionBisection"/>
     <addaction name="actionNewton"/>
    </widget>
    <addaction name="actionInstrukcja"/>
    <addaction name="menuPrzyklad"/>
   </widget>
   <addaction name="menuPomoc"/>
  </widget>
  <action name="actionInstrukcja">
   <property name="text">
    <string>Instrukcja</string>
   </property>
  </action>
  <action name="actionBisection">
   <property name="text">
    <string>Połowienia</string>
   </property>
  </action>
  <action name="actionNewton">
   <property name="text">
    <string>Newtona</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
